contributors.txt

```diff
@@ -136,5 +136,6 @@ YYYY/MM/DD, github id, Full name, email
 2020/07/03, gtisdelle, George Tisdelle, gwnt97(at)gmail.com
 2020/07/07, bstecisa, Borja Serrano, borjaserrano(at)gmail.com
 2020/08/02, FodderMK, Kevin Yockey, gitex{at]foddermk.com
+2020/09/09, adamshakhabov, Adam Shakhabov, theadamo86{at]gmail.com
 2020/09/13, hoborm, Mihaly Hobor, mail{{at]]hobor.xyz
```

Comment:
Standardize the replacement `"(at)"` for the `@` character in all three lines, please.

Reply:
This doesn't actually matter. I did it so it is human readable, but has slightly reduce readability for email harvesters.